spine.csl

```diff
@@ -0,0 +1,197 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="in-text" version="1.0" page-range-format="minimal" demote-non-dropping-particle="sort-only" default-locale="en-US" xmlns="http://purl.org/net/xbiblio/csl">
+  <info>
+    <title>Spine</title>
+    <id>http://www.zotero.org/styles/spine</id>
+    <link href="http://www.zotero.org/styles/spine" rel="self"/>
+    <link href="http://www.zotero.org/styles/sage-vancouver" rel="template"/>
+    <link href="http://edmgr.ovid.com/spine/accounts/ifauth.htm" rel="documentation"/>
```

Comment:
@adam3smith, I really dislike these URLs, because they never have an easy link back to the journal homepage.